torch/csrc/lazy/core/tensor_impl.cpp

```diff
@@ -127,6 +136,10 @@ void LTCTensorImpl::shallow_copy_from(
   generation_ = 0;
 }
 
+c10::SymIntArrayRef LTCTensorImpl::sym_sizes_custom() const {
```

Comment:
i don't understand the convention about sym_sizes_custom and sym_sizes_default; I saw the logic that checks a policy and picks one.

Reply:
yeah, that's pretty much it. I commented a little bit more about it in the comment right above this one.